esmvaltool/recipes/recipe_gier2020bg.yml

```diff
@@ -1,19 +1,26 @@
 # ESMValTool
 # recipe_gier2020bg.yml
-# Diagnostic with growing season temperature requires MODIS Land Cover map in
+# Diagnostic with growing season temperature requires MODIS Land Cover 1 degree map in
 # auxiliary data directory, download from:
 # https://daac.ornl.gov/cgi-bin/dsviewer.pl?ds_id=968 or
 # https://thredds.daac.ornl.gov/thredds/catalog/ornldaac/968/catalog.html
+# NOTE:
+# Commented out datasets and options reflect the exact settings used in the paper creation,
+# datasets commented out require additional preprocessing e.g. converting from hybrid coordinates to
+# pressure coordinates, or are otherwise currently not available on the DKRZ ESGF node.
```

Comment:
```suggestion
# pressure coordinates, or are otherwise currently not available on the DKRZ ESGF node, but may be available on other ESGF nodes, if the user turns on automatic download in the run configuration.
```